benbarnett02.CodeTrack/CodeTrack/Program.cs

```diff
@@ -0,0 +1,310 @@
+﻿using Spectre.Console;
+using System.Globalization;
+namespace TrackingProgram;
+
+// This class handles all user interface elements and input validation.
+class Program
```

Comment:
There's definitely too much going on in your Program class. This should be the starting point of your app. As per the project's requirements (separation of concerns), it would be good to split your code in multiple files with their classes. (i.e. UserInput.cs, MainMenu.cs, Validation.cs). It will be good OOP practice.